app/database/group_attempt_store_compute_all_group_attempts.go

```diff
@@ -101,7 +101,8 @@ func (s *GroupAttemptStore) ComputeAllGroupAttempts() (err error) {
 							MAX(aggregated_children_attempts.validated_at) AS max_validated_at,
 							MAX(IF(items_items.category = 'Validation', aggregated_children_attempts.validated_at, NULL))
 								AS max_validated_at_categories,
-							IFNULL(AVG(IFNULL(aggregated_children_attempts.score, 0) * items_items.score_weight), 0) AS average_score
+							IFNULL(SUM(IFNULL(aggregated_children_attempts.score, 0) * items_items.score_weight), 0) /
```

Comment:
I do not get the outer "IFNULL". `IFNULL(aggregated_children_attempts.score, 0)` cannot be null, `items_items.score_weight`cannot be null, so the sum of the product cannot be null.

Reply:
aggregated_children_attempts.score is null when there are no attempts for the item

Reply:
"the **outer** "IFNULL"

Reply:
fixed